src/resdk/tables/variant.py

```diff
@@ -254,7 +254,18 @@ def _get_obj_geneset(self, obj):
             if gs.output["source"] != "UCSC":
                 qs = self.resolwe.feature.filter(feature_id__in=list(obj_geneset))
                 id_2_name = {obj.feature_id: obj.name for obj in qs}
-                obj_geneset = set([id_2_name[gene] for gene in obj_geneset])
+                obj_geneset = set(
```

Comment:
Do we have a lot of names in obj_geneset? Here we iterate over the list twice, but we could do everything in a single pass.

Reply:
Another remark: creating a list inside a set is not necessary.